profiles/tanv33.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/79161723/d21d81d6-83d2-41cc-85a4-1758095dcc00",
+  "issueId": 1865,
+  "name": "Tanveer Khan",
+  "username": "tanv33"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

I have added endline. Kindly merge the branch. 

Is there anything you want to be done from myside??

Thank you